plugins/schedule/base/lib/iCal/kSchedulingICalComponent.php

```diff
@@ -158,9 +158,9 @@ public function getComponents()
 		return $this->components;
 	}
 
-	public function setField($field, $value)
+	public function setField($field, $value, $subField = null)
 	{
-		$this->fields[strtoupper($field)] = $value;
+		$this->fields[strtoupper($field) . $subField] = $value;

```

Comment:
should we always have upper here as well subField?

Reply:
it's for this case:
DTSTART;TZID=Asia/Jerusalem:20240925T145352
so maybe the naming is a bit misleading. changing to $subFieldVal